.github/workflows/rust.yml

```diff
@@ -148,7 +148,8 @@ jobs:
     steps:
     - uses: actions/checkout@v3
     - uses: stellar/actions/rust-cache@main
-    - run: rustup install nightly
+    # TODO: Upgrade to latest nightly after problem that was introduced in nightly-2024-02-05 (https://github.com/dalek-cryptography/curve25519-dalek/issues/618) is resolved.
+    - run: rustup install nightly-2024-02-03
```

Comment:
It would appear that one of the deps does not build on the latest nightly, `nightly-2024-02-05`. I opened an issue here on the respective lib, although unclear to me if this is an issue with rustc or the lib:
- https://github.com/dalek-cryptography/curve25519-dalek/issues/618